docs/local_forger.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Local Forging Package
+id: local_forger
+author: Michael Kernaghan
+---
+
+# How to Use the Taquito Local Forging Package
+
+If you are looking for a way to interact with the Tezos blockchain, Taquito is a great option. One of the tools offered by Taquito is the local forging package, which allows you to simulate the forging and signing of transactions locally without interacting with a node. Here is a step-by-step guide on how to use the Taquito local forging package:
```

Comment:
> If you are looking for a way to interact with the Tezos blockchain, Taquito is a great option.
I feel this is a bit out of place in here. Anyone who reads the docs for local forging is expected to know what Taquito is.

> which allows you to simulate the forging and signing of transactions
I think "simulate" and "and signing" is better be removed here, and an informal definition of forging can be helpful.

Reply:
i removed the first sentence, but the extra words from the second one